PYME/Acquire/Hardware/HamamatsuDCAM/HamamatsuORCA.py

```diff
@@ -442,3 +442,11 @@ def __init__(self, camNum, multiview_info):
         # default to the whole chip
         default_roi = dict(xi=0, xf=2048, yi=0, yf=2048)
         MultiviewCameraMixin.__init__(self, multiview_info, default_roi, HamamatsuORCA)
+
+
+class MultiviewFusion(MultiviewCameraMixin, Fusion):
```

Comment:
```suggestion
#TODO - replace MultiviewCameraMixin with a Multiview wrapper so that we don't need to #have explicit multiview versions of all cameras.
class MultiviewFusion(MultiviewCameraMixin, Fusion):
```